tests/dragonfly/connection_test.py

```diff
@@ -561,7 +561,7 @@ async def measure(aw):
     e = async_client.pipeline(transaction=True)
     for _ in range(100):
         e.incr("num-1")
-    assert await measure(e.execute()) == 2  # OK + Response
+    assert await measure(e.execute()) == 1  # OK + Response
```

Comment:
We now flush 1 less time in ReplyBuilder. Not sure what the `OK + Response` means here. I tested both on master here and the result of `e.execute()` looks the same when printed via `log.info` :thinking: